src/OrchardCore.Modules/OrchardCore.Search.Lucene/Migrations.cs

```diff
@@ -157,7 +157,7 @@ public int Create()
                 var logger = scope.ServiceProvider.GetService<ILogger<Migrations>>();
                 var tablePrefix = session.Store.Configuration.TablePrefix;
                 var documentTableName = session.Store.Configuration.TableNameConvention.GetDocumentTable();
-
+                
```

Comment:
Remove extra spaces, under Visual Studio you can do ctrl+K+D to format the file.